src/utils/env.ts

```diff
@@ -133,6 +136,9 @@ export const ENV = {
       locales.push(ENV.AUTH_LOCALE_DEFAULT);
     return locales;
   },
+  get AUTH_PROVIDER_SIGNUP_ENABLED() {
```

Comment:
Would you mind adding a short explanation of the environment variable to the [list of env vars](https://github.com/nhost/hasura-auth/blob/main/docs/environment-variables.md)?

Reply:
done